### ISSUE TITLE
Uncaught (in promise) TypeError: Cannot read property 'nodeType' of null

### ISSUE THREAD

**Issue by mnts** [Polymer/lit-html#1047]
already 10hrs tryna understand and find out why this issue happens.. 

  | __commitText | @ | parts.js:55
-- | -- | -- | --
  | commit | @ | parts.js:52
  | update | @ | template-instance.js:18
  | __commitTemplateResult | @ | parts.js:58
  | commit | @ | parts.js:52
  | render | @ | render.js:29
  | render | @ | shady-render.js:118
  | update | @ | lit-element.js:72
  | performUpdate | @ | updating-element.js:192
  | _enqueueUpdate | @ | updating-element.js:173
  | async function (async) |   |  
  | _enqueueUpdate | @ | updating-element.js:170
  | _requestUpdate | @ | updating-element.js:153
  | set | @ | updating-element.js:60
  | (anonymous) | @ | component.js:203
  | (anonymous) | @ | Link_mongo.js:267
  | (anonymous) | @ | User.js:18
  | Promise.then (async) |   |  
  | load | @ | User.js:18
  | (anonymous) | @ | Link_mongo.js:265
  | load | @ | Link_mongo.js:237
  | checkOwner | @ | Link_mongo.js:263
  | (anonymous) | @ | component.js:202
  | load | @ | Link_mongo.js:237
  | load_src | @ | component.js:201
  | attributeChangedCallback | @ | component.js:193
  | attr | @ | jquery.min.js:2
  | _ | @ | jquery.min.js:2
  | attr | @ | jquery.min.js:2
  | k.fn.init | @ | jquery.min.js:2
  | k | @ | jquery.min.js:2
  | (anonymous) | @ | catalogem-items.js:227
  | load | @ | Link_mongo.js:237
  | append | @ | catalogem-items.js:225
  | (anonymous) | @ | catalogem-items.js:254
  | placeLinks | @ | catalogem-items.js:254
  | (anonymous) | @ | catalogem-items.js:209
  | message | @ | ws.js:301
  | connection.onmessage | @ | ws.js:106

**Comment by jridgewell** [comment]
Without a reproduction case, the best I can do is assume you're hitting https://github.com/Polymer/lit-html/issues/293.

**Comment by mnts** [comment]
Im done uing litelement because of this error

**Comment by nboisteault** [comment]
I had this error too and handled it by adding HTML elements in my template which is inside another one : 
```javascript
        const accuracyTemplate = () => html`
            Accuracy (m) : ${mainLizmap.geolocation.accuracy}`;
```
was not working
```javascript
        const accuracyTemplate = () => html`
            <div>Accuracy (m) : ${mainLizmap.geolocation.accuracy}</div>`;
```
works

Hope it will help.

**Comment by justinfagnani** [comment]
> Im done uing litelement because of this error

It really help to have reproductions.

**Comment by jridgewell** [comment]
Can we close this as a dupe of #293 (which is newly fixed by #966)?

**Comment by justinfagnani** [comment]
I suspect it is the same. Closing.
